test/common.js

```diff
@@ -497,6 +497,12 @@ function fail(msg) {
 }
 exports.fail = fail;
 
+exports.mustNotCall = function(msg) {
+  return function dontCall() {
+    fail(msg || 'function should not have been called');
+  };
+};
+
```

Comment:
It could. The whole point is to get beautiful assertion messages though. Using `common.mustCall()` would yield:

`Mismatched <anonymous> function calls. Expected 0, actual 1.`